### ISSUE TITLE
vibranceGUI not working on CS:GO

### ISSUE THREAD

**Issue by imsecrett** [juv/vibranceGUI#35]
I've always played CS:GO and vibrance usually worked... Today I started CS:GO and vibrance wasn't working, any ideas how to fix or what could the problem be?

**Comment by xJoeyv** [comment]
Same here, It doesnt work on Fullscreen CSGO if it is windowed or Fullscreen windowed it works. Didnt test on other games yet

**Comment by imsecrett** [comment]
Mine isn't working in none of them :/

**Comment by juvlarN** [comment]
The new CSGO update broke vibranceGUI. It works for most of the people, but for some it is broken. See here: https://www.reddit.com/r/GlobalOffensive/comments/691j8c/digital_vibrance_option_is_not_applied_to_csgo/ 
Other games should work.

**Comment by imsecrett** [comment]
Is there any fix for it??

**Comment by juvlarN** [comment]
@imsecrett 
1) add another game or add your browser just to test if vibrancegui works
2) set 100% (nvidia) or 200+% (amd) to that game/program you selected
3) set windows vibrance value to something lower
4) tab in + out the program to see if the vibrance value gets changed

**Comment by juvlarN** [comment]
Btw, I am sure that it is not related to vibranceGUI and rather caused by CSGO itself because:
1) Close VibranceGUI completely
2) Go to your NVIDIA / AMD Control Panel
3) Search "Digital Vibrance" option for NVIDIA (Display -> Desktop Color Settings -> 3. Digital Vibrance) or "Saturation" for AMD (My Digital Flat Panels -> Display Color -> Saturation) 
4) Set the value to 50% (NVIDIA) or 100% (AMD)
5) You will see that the vibrancy is back to default value.
6) Set the value to 100% (NVIDIA) or 200% (AMD)
7) You will see that the vibrancy is on the maximum now, in every program and Windows itself. All games get affected by the effect.
8) Start CSGO. You will see that the vibrancy is gone in CSGO. 

I did these steps with 4 guys that complained on Twitter. All of them told me that the vibrance value is not getting registered by CSGO but is visible in any other program/game. So we conclude that this has nothing to do with vibranceGUI and is a problem caused  by the recent CSGO update.

**Comment by imsecrett** [comment]
What I'm asking is, do you have any idea if you can actually make it work on CS:GO again when you find out what's wrong with it?

**Comment by juvlarN** [comment]
Any news on this? Is it working for someone again?

**Comment by Merandy** [comment]
Not working since that May 5, really annoying to play csgo without it. valve has to ruin everything as per usual

**Comment by juvlarN** [comment]
One thing that was weird is that all of the guys on twitter had a NVIDIA driver version that they installed by theirselves. When I updated to Windows 10 Creators Update, a NVIDIA driver version got installed that gets distributed by Microsoft Updates. I can't even find the driver version number I have (382.05) on the NVIDIA website. 

If someone is really bored, one could try to entirely purge the NVIDIA drivers with the ["Display Driver Uninstaller"](http://www.guru3d.com/files-details/display-driver-uninstaller-download.html) tool and see if Windows automatically downloads a new driver version from the Microsoft Update service. I'm not sure if this will solve the problems though.

**Comment by Merandy** [comment]
Just tried it (I have a lot of time on my hands and im desperate), didnt fix the issue, could be just me tho, but at this point im 100% sure the problem is from csgo itself, not my software/hardware, Because VibranceGUI works perfectly on desktop but not in csgo. thanks for trying man you're awesome!

I guess we just gotta get used to CSGO's awful colors

**Comment by JasmuheeN** [comment]
For me vibranceGUI does not work at all . i have a trashy pc and it sais that it cant find my graphicsAdapter type . what can i do ?

**Comment by juvlarN** [comment]
@imsecrett can you try out the latest beta and see if it works now: [vibrancegui.com/vibrance/beta](vibrancegui.com/vibrance/beta)  

@JasmuheeN install your nvidia/amd gpu driver. Remember that Intel and NVIDIA laptops are not supported, as outlined on the website.

**Comment by spartaque12** [comment]
this shit doesnt work for me too , ive no idea how the fuck it works for other ppl

**Comment by xJoeyv** [comment]
@spartaque12 It's Github here not Steam Forums. But you can try out the beta and install the latest drivers. Also switching your Resolution from Fullscreen > Windowed > Fullscreen can help on CSGO

**Comment by juvlarN** [comment]
For everyone having this problem:
1) Shut down vibranceGUI
2) Download beta linked above
3) Navigate to %appdata%\vibranceGUI in your explorer 
4) Delete applicationData.xml
5) Restart vibranceGUI.exe
6) Re-add any application you had added before
7) See if it works

I had two people on Twitter who said that this helped them to fix this issue. 
I am still looking into what could have caused the problem. Would be great if someone can take a look and see if this problem is solved by deleting the applicationData.xml.

**Comment by NicoOsu** [comment]
@juvlarN Hey, I know im pretty late but I hope it may helps you. I done the step by step tutorial you wrote and it worked. After the rename of the .xml file I started VibranceGUI and added CsGo manually. After that I started CsGo and it worked. So for a test, I tabbed out and in and then the vibrance got resetted. I uploaded the old file that I renamed. btw thanks for the help :)
[mimi.zip](https://github.com/juvlarN/vibranceGUI/files/2586002/mimi.zip)
